lib/activities/activity.ex

```diff
@@ -0,0 +1,32 @@
+defmodule GlassFactoryApi.Activities.Activity do
+  @moduledoc """
+  Defines the Activity of an organization.
+  """
+  alias GlassFactoryApi.Activities.Activity
+
+  @type t() :: %Activity{
```

Comment:
Most style guides recommend `__MODULE__` to self-reference the current module:

- https://github.com/christopheradams/elixir_style_guide#module-pseudo-variable
- https://github.com/rrrene/elixir-style-guide#reference-current-module
- https://github.com/lexmag/elixir-style-guide#current-module-reference

This way you can even remove [this alias too](https://github.com/Cendrao/glass_factory_api/pull/47/files#diff-d9c76a7697ef163145d4baf2c94678b1R5).

WDYT?

Reply:
I like that 😄